Libraries/Image/RCTImageStoreManager.m

```diff
@@ -76,13 +76,29 @@ - (void)storeImage:(UIImage *)image withBlock:(void (^)(NSString *imageTag))bloc
 {
   RCTAssertParam(block);
   dispatch_async(_methodQueue, ^{
-    NSString *imageTag = [self _storeImageData:RCTGetImageData(image.CGImage, 0.75)];
+    NSInteger imageOrientation = [self _convertUIImageOrientationToCGImagePropertyOrientation:image.imageOrientation];
+    NSString *imageTag = [self _storeImageData:RCTGetImageData(image.CGImage, 0.75, imageOrientation)];
     dispatch_async(dispatch_get_main_queue(), ^{
       block(imageTag);
     });
   });
 }
 
+- (NSInteger)_convertUIImageOrientationToCGImagePropertyOrientation:(UIImageOrientation)imageOrientation
```

Comment:
It should not be instance method because it completely stateless. It should be `static` C function.
And we should name it something like `CGImagePropertyOrientationFromUIImageOrientation`.
And, also, it would be great if we add comment with this link: https://stackoverflow.com/a/6699649/496389 .
And, moreover, if we create additional enum for all CGImagePropertyOrientation values from documentation (something like that: https://github.com/facebook/react-native/blob/master/React/Base/RCTRootView.h#L23), it would be just awesome!! 😄